build/syntax-highlight.js

```diff
@@ -40,6 +60,7 @@ const loadAllLanguages = lazy(() => {
 const ALIASES = new Map([
   ["sh", "shell"],
   ["jssyntax", "js"],
+  ["idl", "webidl"],
```

Comment:
It is detrimental to allow this:
```suggestion
  // ["idl", "webidl"],  // block list
```

Reply:
Could you elaborate why it is detrimental to allow `idl`/`webidl`?  Once all occurrences have been removed, we can remove it from the list here:

```
% rg '```(web)?idl' content translated-content 
content/files/en-us/mdn/writing_guidelines/howto/write_an_api_reference/information_contained_in_a_webidl_file/index.md
43:```webidl
53:```webidl
65:```webidl
75:```webidl
81:```webidl
114:```webidl
121:```webidl
166:```webidl
179:```webidl
196:```webidl
209:```webidl
217:```webidl
225:```webidl
240:```webidl
251:```webidl
264:```webidl
289:```webidl
322:```webidl
337:```webidl
355:```webidl
365:```webidl
388:```webidl
407:```webidl
415:```webidl
432:```webidl
452:```webidl
473:```webidl
502:```webidl
517:```webidl
533:```webidl
543:```webidl
551:```webidl
560:```webidl
573:```webidl
582:```webidl
590:```webidl

translated-content/files/ja/mdn/writing_guidelines/howto/write_an_api_reference/information_contained_in_a_webidl_file/index.md
579:```webidl

translated-content/files/zh-cn/web/api/pointer_lock_api/index.md
179:```idl
```

Reply:
I think the point is that we don't want to encourage the addition of more webidl, and providing syntax highlighting for it may give contributors the idea that we want to add it.  However, I'm personally thinking the same thing as @caugner -- we add it to the syntax highlighting for now, then remove it when all the instances are taken care of!

Reply:
Correct. WebIDL is a "specification thing", and if you include it in the docs, it is pretty much always an indication that you are doing the wrong thing. 

Please just remove it. The syntax highlighting isn't needed, and adding it here just means that in future we have to _remember_ to remove it.

Reply:
I don't feel strongly about adding `webidl`, so let's just remove it @queengooborg. After all, [Information contained in a WebIDL file](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Write_an_API_reference/Information_contained_in_a_WebIDL_file) is the only article that would legitimately profit from it. 😉